packages/quiz/README.md

```diff
@@ -1,3 +1,55 @@
+## Overview
+
+The `quiz` package lets us add quiz questions to email sequences by linking to the questions from the ends of emails and then triggering the addition of specific tags in ConvertKit.
+
+Each of those tags can then be set up in ConvertKit to trigger the next email in the sequence. (Or anything else tags can do in ConvertKit.) This creates a better learning experience because it only sends the next email at the point someone is ready for it rather than just sending one after another regardless of whether they're ready for it.
+
+When using the quiz questions in emails, it's important to configure ConvertKit to automatically add the `ck_subscriber_id` query string value to all links in the emails. That's how the tags are tied back to the relevant subscriber.
+
+The Process for setting up quiz questions and connecting them in ConvertKit is [currently a work-in-progress and lives in Roam](https://roamresearch.com/#/app/egghead/page/GepByGFy4). It goes into_more details of how to set up the quiz questions and answers and how to wire everything up.
+
+## Quiz Configuration
+
+The top-level quiz configuration can be be found in [src/config.ts](https://github.com/skillrecordings/products/blob/main/packages/quiz/src/config.ts). It controls the product title, author, and the 'after completion messages'. The defaults are generally good, but these can be customized for individual partner products if we need to adjust the tone/voice of the messaging.
+
+## Question & Answer Configuration
+
+### Three Types
+
+- **1.** `essay` will present a `textarea` for the learner to type a response
+- **2a.** `multiple-choice` with one correct answer will present the options as radio buttons
+- **2b.** `multiple-choice` with more than one correct answer will present the options as checkboxes
```

Comment:
this is only two types of question. There are actually 3 in the repository with "multiple choice image" being the third, though these are effectively prebuilt composed example types and functionally this package is about the data types.

here's how a question is composed for example:

```typescript

import * as React from 'react'
import {
  Question,
  QuestionHeader,
  QuestionChoices,
  QuestionBody,
  QuestionFooter,
  QuestionAnswer,
  QuestionSubmit,
  QuestionProps,
} from './index'

const MultipleChoice: React.FC<{
  question: QuestionProps
}> = ({question}) => {
  return (
    <Question {...question}>
      <QuestionHeader />
      <QuestionBody>
        <QuestionChoices />
        <QuestionSubmit>Submit</QuestionSubmit>
        <QuestionAnswer />
      </QuestionBody>
      <QuestionFooter />
    </Question>
  )
}

export default MultipleChoice
```

You can get pretty far with those three types!